homeassistant/components/arcticspa/hottub.py

```diff
@@ -0,0 +1,31 @@
+"""hottub contains the Device class that represents an Arctic Spa Hot Tub."""
+from hashlib import sha256
+from http import HTTPStatus
+from urllib.error import HTTPError
+
+from pyarcticspas import Spa
+
+
+class Device:
```

Comment:
This class should be moved to the lib

Reply:
Good point! The whole class turned out to be unnecessary. Removed.